apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/question/DeleteQuestionService.java

```diff
@@ -0,0 +1,64 @@
+package gov.cdc.nbs.questionbank.question;
+
+import java.util.UUID;
+
+import javax.transaction.Transactional;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.stereotype.Component;
+import org.springframework.stereotype.Service;
+
+import gov.cdc.nbs.questionbank.exception.DeleteException;
+import gov.cdc.nbs.questionbank.kafka.message.DeleteQuestionRequest;
+import gov.cdc.nbs.questionbank.kafka.message.QuestionBankEventResponse;
+import gov.cdc.nbs.questionbank.kafka.message.util.Constants;
+import lombok.RequiredArgsConstructor;
+
+@Service
+@RequiredArgsConstructor
+public class DeleteQuestionService {
+
+	@Value("${kafkadef.topics.questionbank.deletestatus}")
+	private String questionDeleteStatusTopic;
+
+	private final KafkaTemplate<String, DeleteQuestionRequest> kafkaQuestionDeleteTemplate;
```

Comment:
could rename the `DeleteQuestionRequest` to something like `QuestionDeletedEvent` since its no longer a "request" thats looking for something to process it. Its now just a notification that an event took place

Reply:
sure will update.